common/hooks/useFocusTrap.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import { useEffect, type ElementRef } from 'react';
+
+type TrapElRef = {|
+  current: ElementRef<any>,
+|};
+
+export default (startRef: ?TrapElRef, endRef: ?TrapElRef) => {
+  function handleTrapStartKeyDown(event) {
+    if (event.shiftKey && event.keyCode === 9) {
```

Comment:
Is [this](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) better?

Reply:
Yes, almost certainly. In my head it wasn't as well supported as `keyCode` (inspite of `keyCode` being deprecated). But looks like I'm wrong about that. Will update.